src/views/pages/_index.html

```diff
@@ -8,6 +8,34 @@
 <link rel="ractive" href="../components/header-gradient.html" name="c-header-gradient">
 
 <r-page noYield="{{noYield}}" title="{{title}}" description="{{description}}">
+	{{#partial seo}}
+	<script type="application/ld+json">
+		[{
+			"@context":"http://schema.org",
```

Comment:
I can include more people data here such as founders or company size. Should I do that? This particular snippet is the most impactful one out of everything else.

e.g.
![image](https://user-images.githubusercontent.com/38220115/189554507-288cae2f-cd68-4120-a341-90eb15a59f8f.png)

Reply:
I'd say as long as there's a schema for it and the information is at least somewhat meaningful, add it.